Packs/CrowdStrikeFalconSandbox/Integrations/CrowdstrikeFalconSandboxV2/CrowdStrikeFalconSandboxV2.yml

```diff
@@ -2717,5 +2715,5 @@ script:
           type: String
   runonce: false
 tests:
-  - CrowdstrikeFalconSandboxV2 Test
```

Comment:
what is the change here?

Reply:
There is no playbook test which has this name, so i changed it to the same name but without V, 
from CrowdstrikeFalconSandboxV2 Test to -> CrowdstrikeFalconSandbox2 Test
(I checked it with Chanoch and Binat to be sure)